src/main/java/gregtech/common/metatileentities/MetaTileEntities.java

```diff
@@ -710,7 +713,16 @@ public static void init() {
         CREATIVE_CHEST = registerMetaTileEntity(1668, new MetaTileEntityCreativeChest(gregtechId("creative_chest")));
         CREATIVE_TANK = registerMetaTileEntity(1669, new MetaTileEntityCreativeTank(gregtechId("creative_tank")));
 
-
+        // Energy Converter, IDs 1670-1729
+        endPos = GTValues.HT ? ENERGY_CONVERTER[0].length - 1 : Math.min(ENERGY_CONVERTER[0].length - 1, GTValues.UV + 1);
+        int[] amps = {1, 4, 8, 16};
+        for(int i = 0; i < endPos; i++) {
```

Comment:
This will create a ULV converter, which I don't really believe is needed. I am open to other thoughts though.

Reply:
I think its not particularly useful but we generate ULV versions for other electric-type stuff like this so I don't see the harm